iOS/JSQMessagesViewController/build.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
```

Comment:
Is this file necessary as build.cake has not been modified to call it and uses the common.cake.